how-to/manage-users.rst

```diff
@@ -41,14 +41,17 @@ user by running ``sudo -iu stg-launchpad``.
 2. SSH into the ``launchpad-scripts`` juju unit, by running 
 ``in-model qastaging juju ssh launchpad-scripts/leader``.
 
+3. Switch to ``launchpad`` user, by running 
```

Comment:
Is this necessary? I usually go for the `launchpad-admin` unit instead of the `launchpad-scripts` one - both work, but maybe the process in the `scripts` one is different. Afaik within the `launchpad-admin` one there is no need for this.

If so, please update the numbering in the bullet points.

Reply:
I am not entirely sure. Maybe we can update the docs to launchpad-admin. @lgp171188 or @jugmac00 might know better.

Reply:
I am not sure as I have not used this script since we moved away from dogfood.

@tushar5526 Could you just try it out whether this is necessary?

Reply:
@tushar5526 Did you have a chance to try this out?

Reply:
Yes, I think just using launchpad-admin worked sometime back. I think we can just use it.

Reply:
Launchpad-admin just worked for me as well

Reply:
I am okay with documenting this step to be done on the `launchpad-admin` unit as the `ubuntu` user.